lib/Git/Repository/Plugin/GitHooks.pm

```diff
@@ -1562,6 +1562,9 @@ sub grok_acls {
             $acl{action} = $2;
             my $spec     = $3;
 
+            # Interpolate authenticated user. In case user was fetched
+            # by eval, not by env var! See: config item githooks.userenv
+            $spec =~ s/{Git::Hooks-Auth-User}/$git->authenticated_user()/ige;
```

Comment:
I think we don't need to prefix it with Git::Hooks. We just need to use a string which cannot be used as an environment variable. Perhaps just "authenticated-user" or "AUTHENTICATED-USER", since the mere presence of the hyphen would make it a unusable as an environment variable.

What do you think?

Moreover, it's best to document this new behavior of the grok_acls function.

Reply:
Changed to '{AUTHENTICATED-USER}'.